caikit/runtime/service_generation/signature_parsing/parsers.py

```diff
@@ -60,12 +60,6 @@ def get_output_type_name(
                     fn_signature.return_annotation,
                 )
                 return module_class
-
-            log.debug(
-                "Adding %s to list of candidate type names to search for concrete types:",
-                fn_signature.return_annotation,
-            )
-            extra_candidate_names.append(fn_signature.return_annotation)
```

Comment:
Oh shoot did we delete the section where we used `extra_candidate_names`? oops!

I think we'll want a test that does like:
```
def run(stuff) -> "SomeDataModelType":
```
as opposed to
```
def run(stuff) -> SomeDataModelType:
```
and make sure we can still pick up the name from a string

Reply:
(But I would also be open to opening a quick lil bug issue about it if that test fails, and fixing in another PR)